Source/Aptabase/Private/AptabaseAnalyticsProvider.cpp

```diff
@@ -20,10 +20,7 @@ void FAptabaseAnalyticsProvider::RecordExtendedEvent(const FString& EventName, c
 
 bool FAptabaseAnalyticsProvider::StartSession(const TArray<FAnalyticsEventAttribute>& Attributes)
 {
-        int64 EpochInSeconds = FDateTime::UtcNow().ToUnixTimestamp();
-	int Random = FMath::RandRange(0, 99999999);
-	FString RandomString = FString::Printf(TEXT("%08d"), Random);
-	SessionId = FString::Printf(TEXT("%lld%s"), EpochInSeconds, *RandomString);
+	SessionId = FGuid::NewGuid().ToString(EGuidFormats::DigitsWithHyphensLower);
```

Comment:
I do prefer GUIDs but if you prefer the previous ones, I can revert it back.

Reply:
Yeah we need the previous version because being epoch based will enable some important server-side optimizations

Reply:
I've reverted those back now.